flink-table/flink-table-planner/src/main/java/org/apache/calcite/sql/validate/ProcedureNamespace.java

```diff
@@ -61,7 +62,7 @@ public RelDataType validateImpl(RelDataType targetRowType) {
         final SqlOperator operator = call.getOperator();
         final SqlCallBinding callBinding = new FlinkSqlCallBinding(validator, scope, call);
         final SqlCall permutedCall = callBinding.permutedCall();
-        if (operator instanceof SqlWindowTableFunction) {
+        if (operator instanceof SqlWindowTableFunction || operator instanceof SqlMLTableFunction) {
```

Comment:
Calcite have already validated this call at `SqlValidatorImpl#validateCall`, I think we don't need to validate it again.